orocos_kdl/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
 ###################################################
 
 PROJECT(orocos_kdl)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
So in all of the other packages where we do this (like https://github.com/ros2/rclcpp/blob/8cc2a9ba8304b61f240f040bc31414502aaa412c/rclcpp/CMakeLists.txt#L20), we always use a style like:

```
if(NOT CMAKE_CXX_STANDARD)
  set(CMAKE_CXX_STANDARD 14)
endif()
```

Please switch to that.  Once that is done, we can merge this.  Thanks!